BAZProject/Models/MovieDetailSections.swift

```diff
@@ -15,11 +15,11 @@ enum MovieDetailSections: Int {
     var title: String {
         switch self {
         case .cast:
-            return "Elenco"
+            return NSLocalizedString("MDS.cast", comment: "section title")
         case .similar:
-            return "Peliculas similares"
+            return NSLocalizedString("MDS.similarMovies", comment: "section title")
         case .recommendation:
-            return "Peliculas recomendadas"
+            return NSLocalizedString("MDS.recomendationMovies", comment: "section title")
```

Comment:
Just as a comment, a more orderly way to use the NSLocalizedString is by using structs, or enums; you can store the keys and in turn having a function to obtain the String value.



```
struct MovieStrings {
   static let castTitle = "movie_details_cast_title"
}

...
case .cast:
   return MovieStrings.castTitle.localized
...
```